robotbase/src/main/java/com/team2357/frc2022/commands/human/FireTaxiLineCommandGroup.java

```diff
@@ -1,17 +1,19 @@
 package com.team2357.frc2022.commands.human;
 
-import com.team2357.frc2022.commands.feeder.FeederToShooterCommand;
+import com.team2357.frc2022.commands.feeder.FeederShootCommand;
 import com.team2357.frc2022.commands.shooter.ShootTaxiLineCommand;
+import com.team2357.frc2022.commands.shooter.ShooterWaitForRPMsCommand;
 import com.team2357.frc2022.subsystems.FeederSubsystem;
 import com.team2357.frc2022.subsystems.ShooterSubsystem;
 
 import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 
 public class FireTaxiLineCommandGroup extends ParallelCommandGroup{
     public FireTaxiLineCommandGroup () {
-        addCommands(new ShootTaxiLineCommand());
-        addCommands(new FeederToShooterCommand());
 
+        addCommands(new ShootTaxiLineCommand());
+        addCommands(new SequentialCommandGroup(new ShooterWaitForRPMsCommand(), new FeederShootCommand()));
```

Comment:
Should we put ```new SequentialCommandGroup(new ShooterWaitForRPMsCommand(), new FeederShootCommand())``` in it's own command group since we are doing it multiple times?

Reply:
Yeah maybe